code/modules/sanity/inspiration_component.dm

```diff
@@ -33,18 +33,75 @@
+		var/list/adjectives_strong = list("strongly", "heavily", "highly", "naggingly", "unusually")
+		var/list/nouns_medium = list("medium", "moderate", "unremarkable")
+		var/list/adjectives_medium = list("moderately", "relatively", "considerably", "somewhat")
+		var/list/nouns_weak = list("weak", "slight", "mild", "light", "subtle")
+		var/list/adjectives_weak = list("weakly", "slightly", "mildly", "lightly", "subtly")
+		if(prob(75))
+			aspect_noun_or_adjective = "adjective"		
+		switch(stat)
+			if("Robustness")
+				stat_noun = pick("robustness", "strength", "brawls", "fighting")
+				stat_adjective = pick("robust", "strong")
+			if("Toughness")
+				stat_noun = pick("toughness", "resilience", "endurance")
+				stat_adjective = pick("tough", "resilient", "endurant", "stoic")
+			if("Vigilance")
+				stat_noun = pick("vigilance", "watchfulness", "awareness")
+				stat_adjective = pick("vigilant", "watchful", "aware", "wary")
+			if("Mechanical")
+				stat_noun = pick("mechanical", "tools", "crafting", "machinery", "mechanical restoration")
+				stat_adjective = pick("mechanically inclined", "accustomed to machinery", "proficient at mending machinery", "proficient at servicing machinery", "capable of restoring gadgets")
+			if("Biology")
+				stat_noun = pick("biology", "medicine", "medical knowledge", "anatomy", "physiology")
+				stat_adjective = pick("skilled at surgery", "accustomed to anatomy", "knowledgeable in physiology")				
+			if("Cognition")
+				stat_noun = pick("cognition", "intelligence", "knowledge")
+				stat_adjective = pick("smart", "intelligent", "concentrated", "knowledgable")		
 		switch(stats[stat])
 			if(10 to INFINITY)
-				aspect = "an <span style='color:#d0b050;'>overwhelming</span>"
+				a_or_an = "an"
+				stat_color = "<span style='color:#d0b050;'>"
+				if(aspect_noun_or_adjective == "noun")
+					aspect = pick(nouns_overwhelming)
+				else
+					aspect = pick(adjectives_overwhelming)
 			if(6 to 10)
-				aspect = "a <span class='red'>strong</span>"
+				stat_color = "<span class='red'>"
+				if(aspect_noun_or_adjective == "noun")
+					aspect = pick(nouns_strong)
+				else
+					aspect = pick(adjectives_strong)
 			if(3 to 6)
-				aspect = "a <span class='green'>medium</span>"
+				stat_color = "<span class='green'>"
+				if(aspect_noun_or_adjective == "noun")
+					aspect = pick(nouns_medium)
+				else
+					aspect = pick(adjectives_medium)
 			if(1 to 3)
-				aspect = "a <span class='blue'>weak</span>"
+				stat_color = "<span class='blue'>"
+				if(aspect_noun_or_adjective == "noun")
+					aspect = pick(nouns_weak)
+				else
+					aspect = pick(adjectives_weak)
 			else
 				continue
-		to_chat(user, SPAN_NOTICE("This item has [aspect] aspect of [stat]"))
+		if(aspect_noun_or_adjective == "noun")
+			message = pick("This item has [a_or_an] [stat_color][aspect]</span> aspect of [stat_noun]", "[a_or_an] [stat_color][aspect]</span> Aura of [stat_noun] pertains to it")
```

Comment:
```suggestion
			message = pick("This item has [a_or_an] [stat_color][aspect]</span> aspect of [stat_noun]", "Aura of [stat_color][aspect]</span> [stat_noun] pertains to it")
```
So we dont start the message with lower cased a or an
and then dont have Aura capsized in the middle